components/Logo/Logo.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import ussfLogo from 'public/img/ussf-logo.svg'
+
+const Logo = () => {
+  return <img src={ussfLogo} alt="USSF Logo" />
```

Comment:
This url was rendering as [object object] for me -- turns out the svg imports as an object?! Adding `ussfLogo.src` should fix it.

Reply:
omg I was wrestling with this at the end of the day yesterday 😭 THANK YOU